src/bidiMapper/domains/context/context.ts

```diff
@@ -63,6 +63,22 @@ export class Context {
     await this._cdpClient.Runtime.enable();
     await this._cdpClient.Page.enable();
     await this._cdpClient.Page.setLifecycleEventsEnabled({ enabled: true });
+    this._initializeEventListeners();
+  }
+
+  private _initializeEventListeners() {
+    this._handleDomContentEvent();
+  }
+
+  private _handleDomContentEvent() {
+    this._cdpClient.Page.on('domContentEventFired', async (params) => {
+      await this._bidiServer.sendMessage({
+        method: 'browsingContext.domContentLoaded',
```

Comment:
CDP `Page.lifecycleEvent` fires both "DOMContentLoaded" and "load" events. We could possibly use the `frameId` and/or `loaderId` properties to implement the BiDi `navigation` info.

Reply:
Good point, thanks!
Fixed and implemented `browsingContext.load` event.